Index: Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>class Main {\r\n    public static void main(String[] args) {\r\n        system.out.println(\"----WELCOME TO GIT-----\");\r\n    }\r\n}
===================================================================
diff --git a/Main.java b/Main.java
--- a/Main.java	(revision 7c143cdaee82eb5c4c6d800e12ad8bff74e704e5)
+++ b/Main.java	(date 1772316151333)
@@ -1,5 +1,7 @@
 class Main {
     public static void main(String[] args) {
-        system.out.println("----WELCOME TO GIT-----");
+        System.out.println("----WELCOME TO GIT-----");
+
+        System.out.println("Commit a file with in the new branch called feature");
     }
 }
\ No newline at end of file
