Index: Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>class Main {\r\n    public static void main(String[] args) {\r\n        System.out.println(\"----WELCOME TO GIT-----\");\r\n\r\n        System.out.println(\"Commit a file with in the new branch called feature\");\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Main.java b/Main.java
--- a/Main.java	(revision 401b9cc1f9b6d3e825e18a5e4e13d07f7e7e3fe0)
+++ b/Main.java	(date 1772315862608)
@@ -1,7 +1,115 @@
-class Main {
-    public static void main(String[] args) {
-        System.out.println("----WELCOME TO GIT-----");
+// Mini Banking Transaction Analyzer
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.IOException;
+
+public class Main {
+
+
+   public static void main(String[] args) throws IOException {
+
+      BufferedReader bufferedReader =
+            new BufferedReader(new InputStreamReader(System.in));
+
+      String ownersName = "";
+      int transaction = 0;
+      double amount = 0;
+      double totalIncome = 0;
+      double totalExpense = 0;
+      double netBalance = 0;
+      double largestAmount = Double.NEGATIVE_INFINITY;
+      double averageAmount = 0;
+      double totalAmount = 0;
+      double[] transactions;
+      boolean isRunning = true;
+
+      System.out.println("Welcome to Banking Transaction Analyzer");
+
+      while (isRunning) {
+         System.out.println(">>START || EXIT<<");
+         System.out.println("* Type START to make a Transaction");
+         System.out.println("* Type EXIT to stop a Transaction");
+         System.out.print("Enter = ");
+         final String input = bufferedReader.readLine();
+         if ("START".equalsIgnoreCase(input)) {
+            totalIncome = 0;
+            totalExpense = 0;
+            totalAmount = 0;
+            largestAmount = Double.NEGATIVE_INFINITY;
+            averageAmount = 0;
+            netBalance = 0;
+
+            System.out.print("Enter Your Name = ");
+            ownersName = bufferedReader.readLine().trim().toUpperCase();
+            System.out.println("Welcome " + ownersName + "\n");
+
+            System.out.println("Enter Number of Transaction");
+            System.out.print("Enter = ");
+            transaction = Integer.parseInt(bufferedReader.readLine());
 
-        System.out.println("Commit a file with in the new branch called feature");
-    }
+            transactions = new double[transaction];
+            for (int i = 0; i < transactions.length; i++) {
+
+               if (transaction == 0) {
+                  break;
+               }
+
+               System.out.print("Enter Amount = ");
+               amount = Double.parseDouble(bufferedReader.readLine());
+
+               transactions[i] = amount;
+
+
+               if (amount > 0)
+                  totalIncome += amount;
+
+               if (amount < 0)
+                  totalExpense += Math.abs(amount);
+
+               netBalance = totalIncome - totalExpense;
+               largestAmount = Math.max(largestAmount, transactions[i]);
+
+               totalAmount += amount;
+               if (transaction > 0) {
+                  averageAmount = totalAmount / transaction;
+               } else {
+                  averageAmount = 0;
+               }
+            }
+
+            System.out.println("--------------------------------");
+            System.out.println("Total Income = " + totalIncome);
+            System.out.println("Total Expense = " + totalExpense);
+            System.out.println("Net Balance = " + netBalance);
+            System.out.println("Largest Transaction Value = " + (transaction > 0 ? largestAmount : 0));
+            System.out.println("Average Transaction Value = " + averageAmount);
+            System.out.println("--------------------------------");
+
+
+            if (netBalance > 10000) {
+               System.out.println("Excellent " + netBalance);
+            } else if (netBalance > 0) {
+               System.out.println("Profitable " + netBalance);
+            } else if (netBalance == 0) {
+               System.out.println("Break Even " + netBalance);
+            } else {
+               System.out.println("Loss " + netBalance);
+            }
+            System.out.println("--------------------------------");
+         }
+
+         if (!"START".equalsIgnoreCase(input) && !"EXIT".equalsIgnoreCase(input)) {
+            System.err.println("Error = input does not match. Try again.");
+            continue;
+         }
+
+         if ("EXIT".equalsIgnoreCase(input)) {
+            System.out.println("Exiting Transaction. Come Again.");
+            isRunning = false;
+            break;
+         }
+      }
+      bufferedReader.close();
+   }
 }
\ No newline at end of file
Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision 401b9cc1f9b6d3e825e18a5e4e13d07f7e7e3fe0)
+++ /dev/null	(revision 401b9cc1f9b6d3e825e18a5e4e13d07f7e7e3fe0)
@@ -1,10 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
-# Editor-based HTTP Client requests
-/httpRequests/
-# Ignored default folder with query files
-/queries/
-# Datasource local storage ignored files
-/dataSources/
-/dataSources.local.xml
Index: .idea/RoadmapForJava2026.iml
===================================================================
diff --git a/.idea/RoadmapForJava2026.iml b/.idea/RoadmapForJava2026.iml
deleted file mode 100644
--- a/.idea/RoadmapForJava2026.iml	(revision 401b9cc1f9b6d3e825e18a5e4e13d07f7e7e3fe0)
+++ /dev/null	(revision 401b9cc1f9b6d3e825e18a5e4e13d07f7e7e3fe0)
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" inherit-compiler-output="true">
-    <exclude-output />
-    <content url="file://$MODULE_DIR$" />
-    <orderEntry type="inheritedJdk" />
-    <orderEntry type="sourceFolder" forTests="false" />
-  </component>
-</module>
\ No newline at end of file
Index: .idea/modules.xml
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
deleted file mode 100644
--- a/.idea/modules.xml	(revision 401b9cc1f9b6d3e825e18a5e4e13d07f7e7e3fe0)
+++ /dev/null	(revision 401b9cc1f9b6d3e825e18a5e4e13d07f7e7e3fe0)
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectModuleManager">
-    <modules>
-      <module fileurl="file://$PROJECT_DIR$/.idea/RoadmapForJava2026.iml" filepath="$PROJECT_DIR$/.idea/RoadmapForJava2026.iml" />
-    </modules>
-  </component>
-</project>
\ No newline at end of file
Index: .idea/misc.xml
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
deleted file mode 100644
--- a/.idea/misc.xml	(revision 401b9cc1f9b6d3e825e18a5e4e13d07f7e7e3fe0)
+++ /dev/null	(revision 401b9cc1f9b6d3e825e18a5e4e13d07f7e7e3fe0)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectRootManager" version="2" project-jdk-name="25" project-jdk-type="JavaSDK">
-    <output url="file://$PROJECT_DIR$/out" />
-  </component>
-</project>
\ No newline at end of file
Index: .idea/vcs.xml
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
deleted file mode 100644
--- a/.idea/vcs.xml	(revision 401b9cc1f9b6d3e825e18a5e4e13d07f7e7e3fe0)
+++ /dev/null	(revision 401b9cc1f9b6d3e825e18a5e4e13d07f7e7e3fe0)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$" vcs="Git" />
-  </component>
-</project>
\ No newline at end of file
